docs/config.json

```diff
@@ -579,6 +579,11 @@
               "title": "Joining Nodes via Azure",
               "slug": "/management/guides/joining-nodes-azure/"
             },
+            {
+              "title": "Joining Services via Kubernetes ServiceAccount",
```

Comment:
I have a PR to create a dedicated section re: joining services (https://github.com/gravitational/teleport/pull/23869). No big deal which one we merge first, but I thought I'd let you know in case we need to rebase this PR over #23869.